src/styles/calculator.css

```diff
@@ -0,0 +1,67 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: #f5f5f5;
+  height: auto;
+  top: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  overflow: hidden;
+}
+
+.button {
+  background-color: #e0e0e0;
+  height: 3rem;
+  padding: 0.8rem;
+  font-size: 1.5rem;
+  border-radius: 0.5rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+  border: none;
+  outline: none;
+}
+
+.button-container {
+  display: grid;
+  height: 14.5em;
+  gap: 0.5rem;
```

Comment:
- [ ] In order to match the given design let us kindly improve our styling; change `gap: 0.5rem` to `gap: 0.1rem`